src/impl/source/profiles/profileSync.ts

```diff
@@ -131,12 +130,41 @@ export default class ProfileSync extends ProfileActions {
     }
 
     if (profileStatus.deleted && isdelete) {
-      profileStatus.deleted.forEach(file => {
+      profileStatus.deleted.forEach((file) => {
         if (fs.existsSync(file)) {
           fs.unlinkSync(file);
         }
       });
     }
     return Promise.resolve(profileStatus);
   }
+
+  private async reconcileTabs(profileObj: Profile): Promise<void> {
```

Comment:
Could we use the same method from the reconcileProfile class?

Reply:
Didnt want to couple, will move to a better pattern in later refactoring